src/i18n/strings/en_EN.json

```diff
@@ -424,7 +424,6 @@
     "Advanced": "Advanced",
```

Comment:
lots of strings still missing

Reply:
Strings are back 😀

`newTranslatableError` is now specified as a translatable func (see `matrix_i18n_extra_translation_funcs` in the `package.json`)